packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md

```diff
@@ -1,7 +1,17 @@
-To use the Okta CLI:
+From the Okta CLI:
 
 1. Enter the command: `okta apps create`.
-2. Give your app a name.
-3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application is using the tokens solely in server-side code.
-4. Select **Other**.
-5. For the **Login redirect URI**, enter: `http://localhost:8080/login/callback`.
+2. Type your app name.
+3. Select **Single Page App** or **Web** as the **Type of Application**.
+    - **Single Page App**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
+    - **Web**&mdash;for applications using tokens solely in server-side code
```

Comment:
"solely" might be too strong of a word here, it's possible you would send a token to other applications.

Users also might not understand what access tokens are at this point (ideally they shouldn't need to), but this is a bit outside the scope of this PR

Reply:
I've re-worded this description. Please review again. Thanks!